frameworks/pytorch/get_started_mnist_deploy.ipynb

```diff
@@ -6,23 +6,15 @@
    "source": [
```

Comment:
<p>Worth linking to the mentioned notebook "get_started_mnist_train.ipynb"? Looks like it's in the same directory as this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I brought it back to how it was before. If nothing changed with your grammar bot, there will be an error because of the file name.</p>

Reply:
I'm happy that there was no error.